components/home/SectionSubtitle.js

```diff
@@ -22,6 +22,7 @@ SectionTitle.defaultProps = {
   lineHeight: '24px',
   color: 'black.700',
   mb: 3,
+  mt: 0,
```

Comment:
This seems to have an impact on homepage. Is it intended?

| Before | After |
|--------|----|
| ![image](https://user-images.githubusercontent.com/1556356/142995533-10b5a10b-38d6-4aca-abb6-dee3c4fc39aa.png) | ![image](https://user-images.githubusercontent.com/1556356/142995612-3a30d523-74ec-4d8e-bac3-4fcb3bdb85b9.png)

Reply:
Yeah, it is. I followed up with Memo.